Documentation/operations/upgrade.rst

```diff
@@ -493,6 +493,7 @@ Changed Metrics
 * The ``cilium_cidrgroup_policies`` metric has been renamed to
   ``cilium_cidrgroups_referenced`` for better clarity.
 * The ``cilium_cidrgroup_translation_time_stats_seconds`` metric has been disabled by default.
+* The ``cilium_api_limiter_processed_requests_total`` has now label ``return_code`` to specify the http code of the request.
```

Comment:
These are upgrade notes for Cilium 1.15, notes for 1.16 (the next release) are above, so please add this note to the section above :)